src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,15 @@
-export const CommentInfo = () => <>Put the comment here</>;
+export const CommentInfo = ({ comment: { name, email, body } }) => (
+  <div className="CommentInfo">
```

Comment:
don't use this syntax. It's not readable. Fix it in all the cases
```suggestion
export const CommentInfo = ({ comment }) => {
  const { name, email, body } = comment;

  return (
    <div className="CommentInfo">
```